yt/yt/server/lib/exec_node/config.h

```diff
@@ -523,6 +523,10 @@ class TGpuManagerDynamicConfig
 
     NGpu::TGpuInfoSourceConfigPtr GpuInfoSource;
 
+    //! Passed to job proxy as NVIDIA_DRIVER_CAPABILITIES environment variable.
+    //! nvidia-container-runtime specific.
```

Comment:
Let reword it somehow.

For example "This option is specific for nvidia-container-runtime."

Reply:
Fixed